website/docs/theme-classic.md

```diff
@@ -154,3 +154,20 @@ module.exports = {
   },
 };
 ```
+
+### Adding support for new languages
```

Comment:
Maybe shift this into Markdown Features or make a note there.

I think we need to make Markdown Features its own category already. Blogs can also use the markdown features. E.g. admonitions but now we're putting it under docs which suggests that only docs can use it, and is misleading.